src/components/NavigationBar/index.jsx

```diff
@@ -41,115 +47,242 @@ function NavigationBar({ theme }) {
+            />
+            <Typography
+              component={Link}
+              noWrap
+              sx={{
+                color: 'inherit',
+                display: { md: 'flex' },
+                fontWeight: 700,
+                ml: 1
+              }}
+              to={'/'}
+              variant={'h6'}
             >
-              <select
+              {t('title.base')}
+            </Typography>
+
+            <Box sx={{ flexGrow: 1 }} />
+
+            <Hidden mdDown>
+              <Typography
+                component={Link}
+                noWrap
+                sx={{
+                  color: 'inherit',
+                  mr: 3
+                }}
+                to={'/'}
+              >
+                {t('menu.home')}
+              </Typography>
+              <Typography
+                component={Link}
+                sx={{
+                  color: 'inherit',
+                  mr: 3
+                }}
+                to={'/product/list'}
+              >
+                {t('menu.product-list')} &nbsp;&nbsp;
+                <Badge
+                  badgeContent={numberOfProducts()}
+                  color={'error'}
+                />
+              </Typography>
+              {mobileAppUrl && (
+                <Typography
+                  component={'a'}
+                  href={mobileAppUrl}
+                  noWrap
+                  sx={{
+                    color: 'inherit',
+                    mr: 1
+                  }}
+                >
+                  {t('menu.mobile-app')}
+                </Typography>
+              )}
+              <Select
                 id={'language'}
-                name={'language'}
                 onChange={(event) => changeLanguage(event.target.value)}
+                sx={{
+                  '& .MuiOutlinedInput-notchedOutline': {
+                    border: 'none'
+                  },
+                  '& .MuiSelect-icon': {
+                    color: 'common.white'
+                  },
+                  '&.Mui-focused .MuiOutlinedInput-notchedOutline': {
+                    border: 'none'
+                  },
+                  '&:hover .MuiOutlinedInput-notchedOutline': {
+                    border: 'none'
+                  },
+                  color: 'common.white'
+                }}
                 value={i18n.language}
               >
-                <option value={'en-GB'}>🇬🇧 -English</option>
-                <option value={'pt-PT'}>🇵🇹 - Português</option>
-              </select>
-            </NavDropdown>
-          </Nav>
-          <IconButton
-            color={'inherit'}
-            onClick={toggleDarkMode}
-            size={'small'}
-            sx={{
-              '&:hover': {
-                backgroundColor: '#000000'
-              },
-              backgroundColor: '#495057',
-              ml: 1
-            }}
-            variant={'contained'}
+                <MenuItem value={'pt-PT'}>🇵🇹 PT</MenuItem>
```

Comment:
both values are hidden on light mode (white on white) - mobile

Reply:
there as a white background on the box div element that holds the select option that is not updating the background when switching to dark mode